stream-chat-android-client/src/main/java/io/getstream/chat/android/client/socket/ChatSocket.kt

```diff
@@ -57,7 +55,6 @@ internal open class ChatSocket constructor(
     private var eventsParser: EventsParser? = null
     private var socketConnectionJob: Job? = null
     private val listeners = mutableSetOf<SocketListener>()
-    private val eventUiHandler = Handler(Looper.getMainLooper())
```

Comment:
This is useless... the events are already async. This is only making the SDK slower.